packages/block-editor/src/components/caption/index.native.js

```diff
@@ -29,7 +29,7 @@ const Caption = ( { accessible, accessibilityLabel, onBlur, onChange, onFocus, i
 			fontSize={ 14 }
 			underlineColorAndroid="transparent"
 			textAlign={ 'center' }
-			tagName={ '' }
+			tagName={ 'p' }
```

Comment:
I got really curious about this one. how's giving a tag name fixing the alignment? 🤔

and, is it ok to have both at the same time?

> rootTagsToEliminate={ [ 'p' ] }

> tagName={ 'p' }

Reply:
I didn't have time to do a full investigation, but main main guess, is that when you don't define a tag, the default you get is div, and the Aztec iOS for some reason is ignoring the text alignemnt settings or defaulting the div to left alignment.

The rootTagsToEliminate is just to give the HTML back clean of <p> know that we are using the tagName of p we should no longer need it, because the component already eliminates automatically the tag refered in tagName.

Reply:
I haven't verified but suspicious that [this](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1228) might have caused a regression about this.

Reply:
> I haven't verified but suspicious that this might have caused a regression like this.

@cameronvoell has reached the same conclusion last night so, I guess that's a form of verification.